### ISSUE TITLE
Full URL output

### ISSUE THREAD

**Issue by wvxwxvw** [stephanmahieu/formhistorycontrol-2#92]
Is it possible to display the full url of the record in the windows history not in a column, but separately, such as the status bar? When you hover the mouse pointer over it (the best option), or at least when you select a record.

**Comment by stephanmahieu** [comment]
I think that can be done but lots of people find those kind of tooltips annoying, so that would mean yet another preference. Only show the value of the column when you single-click it would be my preferred solution.

**Comment by stephanmahieu** [comment]
Wait a minute, hovering over the URL does already show a tooltip with the full URL!
Does that mean it is currently not displayed in your case?

**Comment by wvxwxvw** [comment]
No, I meant making it easier to quickly search the list. The hint in the URL column will not help here, as it will take a long time. The column itself will not help either, since the space is limited. But if it was when you hold the cursor over the list and in real time would display a hint in a certain field or at the bottom of the addon window, this is what I had in mind.

**Comment by Chealer** [comment]
@wvxwxvw: As this is an ITS, each ticket should emphasize the problem. Please indicate at least 1 affected version, describe your use case and retitle with a short description of the issue.
